src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -275,7 +275,7 @@ int64_t get_offset(const cldnn::layout& l, dnnl::memory::desc&& desc) {
 
 std::tuple<dnnl::memory::desc, dnnl::memory::desc, dnnl::memory::desc>
 get_conv_memory_descs(cldnn::layout input_layout, cldnn::layout weights_layout, cldnn::layout output_layout, dnnl::memory::format_tag target_fmt) {
-    mem_flags flags = (input_layout.format.is_blocked() || output_layout.format.is_blocked()) ?
+    mem_flags flags = (input_layout.format.is_blocked() || output_layout.format.is_blocked() || format::is_grouped(weights_layout.format)) ?
```

Comment:
This condition is a bit difficult to understand. Maybe should we change `need_blocked` to `use_original_layout`? Or what about introducing an additional flag?

Reply:
Added an additional flag "grouped".